docs/conf.py

```diff
@@ -22,5 +22,8 @@
 html_static_path = ['_static']
 
 
+intersphinx_mapping["guide"] = ('https://cratedb-guide--29.org.readthedocs.build/', None)
```

Comment:
Of course, 7098972c1c52 needs to be removed again before merging. It is only here to demonstrate everything fits together well, by making CI succeed.

Reply:
The CrateDB Guide has been published, and this workaround has been removed.